conda-store-server/conda_store_server/orm.py

```diff
@@ -190,15 +190,22 @@ def build_path(self, conda_store):
         """
         store_directory = os.path.abspath(conda_store.store_directory)
         namespace = self.environment.namespace.name
-        name = self.specification.name
```

Comment:
Not used in `build_directory`.

Reply:
How come? Not anymore? I thought it had the hash, timestamp and name? Or is that part of the 'build_key'  thingy?

Reply:
It was not used before this PR, just a mistake that I fixed since I was making changes here anyway.

This is the one used below, no `name` here:

```py
    build_directory = Unicode(
        "{store_directory}/{namespace}",
        help="Template used to form the directory for storing conda environment builds. Available keys: store_directory, namespace, name. The default will put all built environments in the same namespace within the same directory.",
        config=True,
    )

```

This is the one you're talking about (I don't touch this one):
```py
    environment_directory = Unicode(
        "{store_directory}/{namespace}/envs/{name}",
        help="Template used to form the directory for symlinking conda environment builds. Available keys: store_directory, namespace, name. The default will put all environments in the same namespace within the same directory.",
        config=True,
    )
```